components/raftstore-v2/src/operation/command/write/simple_write.rs

```diff
@@ -21,12 +22,14 @@ pub struct SimpleWriteEncoder {
     buf: Vec<u8>,
     channels: Vec<CmdResChannel>,
     size_limit: usize,
+    likely_proposed: bool,
```

Comment:
This parameter seems is same as "call_proposed_on_success"---its usage basically is call notify_proposed . I think changing the name to "call_proposed_on_success" is more clear.